Lambdas/GHHooks/Extensions/String+Document.swift

```diff
@@ -71,6 +80,21 @@ extension String {
         return lines.joined(separator: "\n")
     }
 
+    /// Doesn't count markdown attributes towards the limit.
+    @_disfavoredOverload
```

Comment:
Why is this attribute needed? (Disfavored overloads are notoriously finicky at best.)

Reply:
The function names are the same.
Unless you can suggest a good function name for the functions that return both the remaining and the result, then we'll need to roll with this.

Reply:
I've basically duplicated the `unicodesPrefix()` function, one just returns the result, the other returns the remaining too.